### PR TITLE
@mzikherman - Adds collaborative filter lab test to related works section

### PR DESCRIPTION
**Pull request by broskoski**
Re: https://github.com/artsy/metaphysics/pull/396

screenshot 2016-09-01 17 25 51

### CODE REVIEW

File: apps/artwork/components/related_artworks/index.jade [hunk 1]
```diff
@@ -17,6 +17,13 @@ if artwork.layer && artwork.layer.artworks && artwork.layer.artworks.length
           )
             = layer.name
 
+        if artwork.related.length && user && user.hasLabFeature('Others also viewed')
```
Comment (mzikherman) [inline apps/artwork/components/related_artworks/index.jade]: if `artwork.related` is null might this crash?

`if artwork.related?.length && user?.hasLabFeature('Others also viewed')`  i might be wrong tho

File: apps/artwork/components/related_artworks/query.coffee [hunk 2]
```diff
@@ -12,5 +12,8 @@ module.exports = """
         ... artwork_brick
       }
     }
+    related(size: 1) {
```
Comment (mzikherman) [inline apps/artwork/components/related_artworks/query.coffee]: Is this just here to check if there are any related works (in order to know whether to render the tab)?

if so, maybe there should be a counts thing in metaphysics for this? not sure what the best practice is for that. #minortho

**Comment by mzikherman** [comment]
Looks great! I made two minor comments that I'm not even sure about, feel free to respond or merge as-is! woo

File: apps/artwork/components/related_artworks/index.jade (hunk 1 above)
Comment (broskoski) [inline apps/artwork/components/related_artworks/index.jade]: good call!

File: apps/artwork/components/related_artworks/query.coffee (hunk 2 above)
Comment (broskoski) [inline apps/artwork/components/related_artworks/query.coffee]: good point, I can follow up!

**Pull request merged**